tests/test_frameworks.py

```diff
@@ -233,6 +236,64 @@ def test_tensorflow_rarelabel(batch_size, data=DATA_RARE_LABEL, hidden_units=8):
     preds = cl.predict(dataset_tf)
 
 
+def test_keras_sklearn_compatability(data=DATA, hidden_units=128):
+    # test pipeline on Sequential API
+    model = KerasWrapperSequential(
+        [
+            tf.keras.layers.Dense(128, input_shape=[data["num_features"]], activation="relu"),
```

Comment:
If we are not testing accuracy of this model, then make it much smaller so the test runs faster? Eg. 2 hidden units instead of 128?